client/src/components/ManageAccount/ManageAccountTables.module.css

```diff
@@ -0,0 +1,15 @@
+.change-password-link {
+  composes: c-button from '../../styles/components/c-button--new.css';
+  composes: c-button--size-small from '../../styles/components/c-button--new.css';
+  composes: c-button__text from '../../styles/components/c-button--new.css';
+  composes: c-button--as-link from '../../styles/components/c-button--new.css';
+}
+
+.edit-profile-link {
+  composes: c-button from '../../styles/components/c-button--new.css';
+  composes: c-button__text from '../../styles/components/c-button--new.css';
+  composes: c-button--as-link from '../../styles/components/c-button--new.css';
+  font-size: 14px;
+  font-weight: bold;
+  padding: 0;
+}
```

Comment:
@wesleyboar  is this pattern using `composes` the right way for using cep styles.

Reply:
It is, for core styles that have not been integrated into a suitable component, as is the case here.

<sub>**Note to Self**: The Core-Portal has a `<Button>` component, that supports all of `c-button`. But `<Button type="link">` is inadequate, semantically, because it creates a `<button>`, but this use case calls for a `<a>`. This is a difficulty that `<Button>` is currently inadequate to resolve.</sub>